src/main/webapp/WEB-INF/web.xml

```diff
@@ -6,13 +6,19 @@
   <display-name>Archetype Created Web Application</display-name>
 
   <servlet>
-    <servlet-name>spring</servlet-name>
-    <servlet-class>org.springframework.web.servlet.DispatcherServlet</servlet-class>
+    <servlet-name>dispatcher</servlet-name>
+    <servlet-class>
+      org.springframework.web.servlet.DispatcherServlet
+    </servlet-class>
+    <init-param>
```

Comment:
What does this line do?

Reply:
It is configuring the dispatcher servlet in our application which will take care of every request sent by user.